### ISSUE TITLE
How to reallize 3D CNN with Matconvnet to learn spatio-temporal Features from Video sequence?

### ISSUE THREAD

**Issue by tzczsq** [vlfeat/matconvnet#359]
Hi,  I am interested in that how to use the existed MatConvnet to realize 3D-CNN to learn spatio-temporal Features from Video sequence, as described in following papers? the key step is how to realize 3D-Convoution and 3D-pooling.
3D convolutional neural networks for human action recognition-2013-ICMI;
Learning Spatiotemporal Features with 3D Convolutional Networks_2015-ICCV
Could you please give an answer?

**Comment by bazilas** [comment]
Hello,

At the comment the library doesn't support convolutions for 3D volumes (X,Y,Z + Channels).

V

**Comment by vedaldi** [comment]
Yes. that's unfortunately correct. However, we are now looking at adding this possibility.
